generators/app/index.js

```diff
@@ -32,13 +32,11 @@ class MxGenerator extends Generator {
          * Test if a widget folder was passed by argument and try to create the new folder
          */
         if(args.length > 0){
-            const dir = args.map(arg => arg.replace(/(^|\s)\S/g, l => l.toUpperCase())).join("");
+            const dir = args.map(arg => arg.replace(/(^|\s)\S/g, l => l.toLowerCase())).join("-");
+            const name = args.map(arg => arg.replace(/(^|\s)\S/g, l => l.toUpperCase())).join("");
```

Comment:
IMO: the name in the package.json name should be qual to the dir name.

Reply:
Fixed